intermediate_notebooks/examples/custreamz/parsing_haproxy_logs.ipynb

```diff
@@ -0,0 +1,427 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parsing HAProxy Logs\n",
+    "\n",
+    "This notebook demonstrates real world log parsing examples. For this notebook we will use the popular HAProxy\n",
+    "format.\n",
+    "\n",
+    "This notebook will cover\n",
+    "+ Streaming log entry event level logs from Kafka using Streamz\n",
+    " - Those interested in Kafka can refer to: https://kafka.apache.org/quickstart to start a local Kafka cluster.\n",
```

Comment:
Is this now automated in a Dockerfile? If so, perhaps that should be linked/explained here?